app/controllers/master/base_controller.rb

```diff
@@ -1,6 +1,8 @@
 class Master::BaseController < ApplicationController
   before_action :force_master_domain
 
+  include FlashAlerts
```

Comment:
I tried changing the base of this controller to `FrontendController`, but it broke a lot of stuff.
Extracting the logic to a module, and reusing it where need is cleaner anyway, I think.